### PR TITLE
readme

### PR DESCRIPTION
**Pull request by NathanSegerlind**
Initial README.md for oni-lda-c version 1.0.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,195 @@
...
+
+oni-lda-c requires an MPI installation on the cluster. It has been validated with [mpich](http://www.mpich.org/) and [intel-mpi](https://software.intel.com/en-us/intel-mpi-library). You must install your favorite MPI on your cluster following the instructions provided by that
+distribution.
+
+### Compilation
+
+Compilation requires that MPI libraries have been installed on the machine where oni-lda-c will be compiled. Other than that, just make in the top oni-lda-c directory:
+```
+[user@system ~]$ cd oni-lda-c
+[user@system  oni-lda-c]$ make
```
Comment (rabarona) [inline README.md]: Current wiki indicates to run make clean before make, is make clean not required anymore? 
https://github.com/Open-Network-Insight/open-network-insight/wiki/Install%20and%20Configure%20ML

File: README.md [hunk 2]
```diff
@@ -1,2 +1,195 @@
...
+
+## oni-lda-c output
+
+
+There are two files that will appear in the output directory:
+
+**final.beta** in which each row corresponds to a topic and each row contains the natural logarithm of the probability of each word given that topic.
+
+For example,  if in topic 1, the probability of word 1 were 0.1, the probability of word 2 were 0.89 and the probability of word 3 were 0.01, then
+then the first line of final.beta would be  −2.302585093 −0.116533816 −4.605170186
```
Comment (rabarona) [inline README.md:176]: "then" written twice.

**Comment by rabarona** [comment]
Very clear and complete.

**Comment by NathanSegerlind** [comment]
thanks, man

File: README.md [hunk 3]
```diff
@@ -1,2 +1,204 @@
-# oni-lda
-LDA parallel version for OpenNetworkInsight
+# oni-lda-c
+
+Parallel implementation of latent Dirchlet allocation for OpenNetworkInsight, version 1.0.1
+
+oni-lda-c is a program for *topic modelling*:  From a collection documents and integer k, infer k topics and assign to each document a mixture of topics and to each topic a mixture of words.
```
Comment (natedogs911) [inline README.md]: should this be "From a collection _of_ documents"?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,204 @@
-# oni-lda
-LDA parallel version for OpenNetworkInsight
+# oni-lda-c
+
+Parallel implementation of latent Dirchlet allocation for OpenNetworkInsight, version 1.0.1
+
+oni-lda-c is a program for *topic modelling*:  From a collection documents and integer k, infer k topics and assign to each document a mixture of topics and to each topic a mixture of words. 
+
+Topic modelling is used in the OpenNetworkInsights suspicious connections analysis to generate profiles of typical traffic on the network and estimate
```
Comment (natedogs911) [inline README.md]: "OpenNetworkInsights" should be change to "Open-Network-Insight"

File: README.md (hunk 3 above)
Comment (NathanSegerlind) [inline README.md]: yes, yes it should be "a collection **of** documents"

File: README.md [hunk 5]
```diff
@@ -1,2 +1,204 @@
...
+the probability of a given communication occurring between two IPs.  Network traffic on a particular channel corresponds to a "document" and
+summaries of the traffic are the "words". 
+
+For example, in a netflow analysis the document is communication between two IP addresses and
+and the "words" are summaries of the netflow records between the two. The
+"topics" are then profiles common traffic on the network, the mixture of topic to the documents
```
Comment (natedogs911) [inline README.md]: profiles of common traffic

**Comment by natedogs911** [comment]
i actually found it very enlightening, perhaps there could be a coffee table paper edition?

**Comment by NathanSegerlind** [comment]
we need more full page color photographs for the coffee  table version

**Pull request merged**